render-only-sample/tests/story-board-app/MainPage.xaml.vb

```diff
@@ -0,0 +1,367 @@
+﻿' The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+Imports Windows.Graphics.Imaging
+Imports Windows.Storage
+Imports Windows.Storage.Streams
+Imports Windows.UI.Xaml.Media.Animation
+''' <summary>
+''' An empty page that can be used on its own or navigated to within a Frame.
+''' </summary>
+Public NotInheritable Class MainPage
+    Inherits Page
+    Dim WithEvents entryStoryBoard As New Windows.UI.Xaml.Media.Animation.Storyboard
+    Dim WithEvents exitStoryBoard As New Windows.UI.Xaml.Media.Animation.Storyboard
+    Dim panX As DoubleAnimation
+    Dim panY As DoubleAnimation
+    Dim panTrans As TranslateTransform
+    Dim scaleAnimatiionX As DoubleAnimation
+    Dim scaleAnimatiionY As DoubleAnimation
+    Dim scaleTrans As ScaleTransform
+    Private Sub MainPage_Loaded(sender As Object, e As RoutedEventArgs) Handles Me.Loaded
+        loadImage() '
+    End Sub
+    Public Async Sub loadImage()
+        'Try
+
+        Dim ctr As Int16 = 1
+
+        LayoutRoot.Resources.Add("st", entryStoryBoard)
+        LayoutRoot.Resources.Add("st1", exitStoryBoard)
+
+        While True
+            Try
+
+
+                Dim str As StorageFile = Await Windows.Storage.ApplicationData.Current.LocalFolder.GetFileAsync("IMG_0435.JPG")
```

Comment:
Hello,
I have commited changes by adding image to assets and changing code accordingly, kindly consider this for pulling. Sorry for inconvenience.